torch_dvf/models/architectures.py

```diff
@@ -0,0 +1,45 @@
+import torch
+
+from torch_dvf.data import Data
+from torch_dvf.nn.gnn import interp, pool
+
+
+class PointNet(torch.nn.Module):
```

Comment:
Let's call it differently, since `PointNet` class already exists in `pointnet.vanilla` and it might be confusing. Maybe `PointNetBase` (I like this better) or `AbstractPointNet`.